pkgs/tools/misc/termtosvg/default.nix

```diff
@@ -0,0 +1,26 @@
+{ lib, buildPythonApplication, fetchFromGitHub, svgwrite, pyte, mock, unittest2 }:
```

Comment:
Please pass only `lib`, `python3` and `fetchFromGitHub` as arguments so that the Python package set can be easily overriden.